Doc/c-api/long.rst

```diff
@@ -571,6 +571,42 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
    .. versionadded:: 3.14
 
 
+.. c:function:: int PyLong_IsPositive(PyObject *obj)
+
+   Check if the integer object *obj* is positive.
```

Comment:
```suggestion
   Check if the integer object *obj* is strictly positive.
```

Maybe add:

```
   Check if the integer object *obj* is strictly positive: if *obj* > ``0``.
```

Reply:
I think there is no ambiguity.  Positive is for, well, being strictly positive.  Else we would use something like "nonnegative".

Reply:
For me, zero is a positive number. "strictly positive" is used in a few places in Python:

```
Modules/_bz2module.c:    /* data_size is strictly positive, but because we repeatedly have to
Modules/_io/bufferedio.c:            "buffer size must be strictly positive");
Modules/_io/textio.c:                        "a strictly positive integer is required");
Modules/zlibmodule.c:    /* data_size is strictly positive, but because we repeatedly have to
Python/gc.c:   objects that can be reached directly from outside must have strictly positive
Python/gc.c:   by the reachable objects (the ones with strictly positive reference count).
Python/sysmodule.c:                        "switch interval must be strictly positive");
```

An alternative is to say "Check if the integer is greater than zero".

Reply:
> For me, zero is a positive number.

Uh, it doubt I'm able to find math source with this meaning.  But a lot for a different opinion:
https://mathworld.wolfram.com/PositiveInteger.html
https://mathworld.wolfram.com/PositiveNumber.html
https://en.wikipedia.org/wiki/Integer

and
```
$ git grep positive Modules/mathmodule.c Modules/cmathmodule.c|wc -l
11
```

There is another story: whether ``0`` is a *natural* number or not.

Reply:
I've applied this advice to positive and negative. Maybe that's more appropriate. Maybe `victor` meant it more purely (it's very positive💯).

Reply:
IMO, this *is* unambiguous, but if you skim this with an expectation of “*sign bit unset*”, it's easy to not notice. So, I too would prefer being extra clear.

```suggestion
   Check if the integer object *obj* is strictly positive (``obj > 0``).
```

Reply:
@rruuaanng: both Victor and Petr have expressed a clear with to include the word "strictly". Please consider their advice.